### PR TITLE
[JA] translation for Expanding Privacy Sandbox Testing

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Japanese translation for the Expanding Privacy Sandbox Testing article. 
PEMT by Alconost

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
CC: @agektmr

File: site/ja/blog/expanding-privacy-sandbox-testing/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+さらに、次の 2 つの**アトリビューションレポート** セッションが予定されています。
+
+*   英語 - 8 月 11 日木曜日 @ 15:00-16:30 GMT
+*   日本語 - 8 月 19 日金曜日 @ 10:00-11:30 JST (01:00-02:30 GMT)
+
+詳細については、[attribution-reporting-api-dev メーリングリスト](https://groups.google.com/a/chromium.org/g/attribution-reporting-api-dev/c/s3QYro6SjeE/m/R6jI9TseAgAJ)のお知らせをご覧ください。
+
+{% endAside %}
+
+また、 **[GitHub に発者サポートリポジトリ](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support)** も用意されています。 イシューが発生した場合や、質問をどこに問題を提起すべきか分からない場合は、 イシューを投稿すると、当社が回答を行ったり、照会する適切な場所を見つけるお手伝いをしたりします。
```
Comment (agektmr) [inline site/ja/blog/expanding-privacy-sandbox-testing/index.md:128]:
```suggestion
また、 **[GitHub に開発者サポートリポジトリ](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support)** も用意されています。 イシューが発生した場合や、質問をどこに問題を提起すべきか分からない場合は、 イシューを投稿すると、当社が回答を行ったり、照会する適切な場所を見つけるお手伝いをしたりします。
```

File: site/ja/blog/expanding-privacy-sandbox-testing/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+フィードバックのハイライト:
+
+*   **Yahoo! Japan** 初期のアトリビューションレポートテストの結果に関する[ホワイトペーパー](https://github.com/WICG/attribution-reporting-api/issues/201)。
+*   [Topics](https://support.google.com/admanager/answer/12270543) オリジントライアルに関する **Google Ad Manager** 初期計画。
+
+{% endAside %}
+
+**[フィードバックページでは、さまざまなルートのそれぞれ](/docs/privacy-sandbox/feedback/)** と、各 API のアクティブなルートについて説明します。 また、[フィードバック フォーム](/docs/privacy-sandbox/feedback/#feedback-form)経由で当社に直接フィードバックを提供することもできます。
+
+最終的に、Cookie の動作を変更することで、28 年間 Web の一部であった技術を変更しています。 Web は私たち全員に属しています。これらの変更に取り組み、よりプライベートな環境を可能にしながら、誰にとっても魅力的な豊かでオープンなエコシステムを可能にする理想的なミックスを追求するには、皆さんからの情報入力と指示が必要です。 この計画にご協力ください。
```
Comment (agektmr) [inline site/ja/blog/expanding-privacy-sandbox-testing/index.md:147]:
```suggestion
最終的に、Cookie の動作を変更することで、28 年間 Web の一部であった技術を変更します。 Web は私たち全員に属しています。これらの変更に取り組み、よりプライベートな環境を可能にしながら、誰にとっても魅力的な豊かでオープンなエコシステムを可能にする理想的なミックスを追求するには、皆さんからの情報と方向性の提供が必要です。 この計画にご協力ください。
```

**Review by agektmr — CHANGES_REQUESTED** [review]

**Comment by ilyaspiridonov** [comment]
@agektmr requested changes integrated, thank you

**Review by agektmr — APPROVED** [review]
LGTM

**Comment by agektmr** [comment]
@isalconost Thank you!

**Pull request merged**